yaml/shared/src/test/scala/org/virtuslab/yaml/parser/AnchorSpec.scala

```diff
@@ -158,7 +141,19 @@ class AnchorSpec extends BaseYamlSuite:
 
     val expectedEvents = List(
       StreamStart,
-      DocumentStart(),
+      DocumentStart(explicit = true),
+      MappingStart(),
+      Scalar("hr"),
+      SequenceStart(),
+      Scalar("Mark McGwire"),
+      Scalar("Sammy Sosa", metadata = NodeEventMetadata(Anchor("SS"))),
+      SequenceEnd,
+      Scalar("rbi"),
+      SequenceStart(),
+      Alias(Anchor("SS")),
```

Comment:
We may want to change the anchor name from `SS` to something else. It's better to avoid such abbreviations, even when example comes from the yaml website